app/build.gradle

```diff
@@ -27,17 +27,19 @@ dependencies {
     androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
         exclude group: 'com.android.support', module: 'support-annotations'
     })
-    compile 'com.android.support:appcompat-v7:26.+'
+    //noinspection GradleCompatible
+    compile 'com.android.support:appcompat-v7:26.1.0'
     compile 'com.android.support.constraint:constraint-layout:1.0.2'
     compile 'com.android.support.constraint:constraint-layout-solver:1.0.2'
     compile 'com.google.firebase:firebase-database:11.4.2'
     compile 'com.google.firebase:firebase-core:11.4.2'
     compile 'com.google.firebase:firebase-auth:11.4.2'
     compile 'com.google.firebase:firebase-storage:11.4.2'
-    compile 'com.android.support:design:26.+'
-    compile 'com.android.support:recyclerview-v7:23.1.1'
-    compile 'com.android.support:support-v4:26.+'
-    compile 'com.android.support:cardview-v7:21.0.3'
+    compile 'com.android.support:design:26.1.0'
```

Comment:
I think we should keep the +'s wherever possible, as otherwise we don't get the bugfixes automatically.

Reply:
I don't exactly feel as strongly as this guy in the following article: http://blog.danlew.net/2015/09/09/dont-use-dynamic-versions-for-your-dependencies/ But he explains why I removed the +. What do you think?

Reply:
I guess his reasoning is pretty solid, namely "we assume that libraries follow proper versioning".
In our case, we can really do either since it's still a small-scale project. I don't really mind either.